src/App.vue

```diff
@@ -3,42 +3,43 @@
     <!-- Dialogs -->
     <dashboard-unavailable-dialog
       :dialog="dashboardUnavailableDialog"
-      @exit="onClickExit"
-      @retry="onClickRetry"
+      @exit="onClickExit()"
+      @retry="onClickRetry()"
       attach="#app"
     />
 
     <business-auth-error-dialog
       :dialog="businessAuthErrorDialog"
-      @exit="onClickExit"
-      @retry="onClickRetry"
+      @exit="onClickExit()"
+      @retry="onClickRetry()"
       attach="#app"
     />
 
     <name-request-auth-error-dialog
       :dialog="nameRequestAuthErrorDialog"
-      @exit="onClickExit"
-      @retry="onClickRetry"
+      @exit="onClickExit()"
+      @retry="onClickRetry()"
       attach="#app"
     />
 
     <name-request-invalid-dialog
       :dialog="nameRequestInvalidDialog"
       :type="nameRequestInvalidType"
-      @exit="onClickExit"
-      @retry="onClickRetry"
+      @exit="onClickExit()"
+      @retry="onClickRetry()"
       attach="#app"
     />
 
     <!-- Initial Page Load Transition -->
-    <transition name="fade">
+    <v-fade-transition>
       <div class="loading-container" v-show="showLoadingContainer">
+        <!-- just show blank page for forced spinner -->
         <div class="loading__content">
           <v-progress-circular color="primary" size="50" indeterminate />
-          <div class="loading-msg" v-if="!isSignoutRoute">Loading Dashboard</div>
+          <div class="loading-msg" v-if="!isSignoutRoute && !forceSpinner">Loading Dashboard</div>
```

Comment:
I changed the property name as it seems a shame to hijack this spinner just for a specific need. This is more general.

Reply:
Obsolete.